src/Morphir/IR/SDK/Result.elm

```diff
@@ -21,8 +21,8 @@ moduleSpec =
         Dict.fromList
             [ ( [ "result" ]
               , CustomTypeSpecification [ [ "e" ], [ "a" ] ]
-                    [ ( [ "ok" ], [ ( [ "value" ], Type.Variable [ "a" ] () ) ] )
-                    , ( [ "err" ], [ ( [ "error" ], Type.Variable [ "e" ] () ) ] )
+                    [ ( [ "ok" ], [ ( [ "value" ], Type.Variable () [ "a" ] ) ] )
+                    , ( [ "err" ], [ ( [ "error" ], Type.Variable () [ "e" ] ) ] )
```

Comment:
Can we use the helper functions here as well?

Reply:
Not sure if that would make sense since nobody will ever refer to `Ok` or `Err` on the type level. The difference is these are `Type.Constructor` values while user code will only ever use `Value.Constructor` (and even that would only happen if you are generating it explicitly).

Reply:
Agreed. I did not read the context of the change.